server/src/server.ts

```diff
@@ -353,7 +353,7 @@ async function validateTextDocument(textDocument: TextDocument, lintOn: Lint) {
                         result.splice(i, 1);
                     }
 
-                    diagnostics = _.uniqBy(diagnostics, function (e) { return e.line + ":::" + e.message; } );
```

Comment:
Here, If I'm not mistaken, `e` should have have a field called `line`

Reply:
I may have had it long ago... or it was just a latent bug. I'm not sure which.

Reply:
Actually, it seems it should be present... At least from the generation code...

Reply:
It's a bug... for sure!

Reply:
Thanks!